BoxContentSDK/BoxContentSDKTests/NSStringBoxContentSDKAdditionsTests.m

```diff
@@ -0,0 +1,38 @@
+//
+//  NSStringBoxContentSDKAdditionsTests
```

Comment:
moved this away from BoxCore since the tested method live in the content sdk now.